docs/docs/training-data-format.mdx

```diff
@@ -254,7 +254,7 @@ In training examples, entity annotation would look like this:
 nlu:
 - intent: check_balance
   examples: |
-    - how much do I have on my [savings]("account") account
+    - how much do I have on my [savings](account) account
```

Comment:
out of scope for this, maybe this is something we can warn about when we validate training data? do we already have a ticket for this?